### PR TITLE
[Frontend] remove angular server, replaced with static files 

### PR DESCRIPTION
**Pull request by AC-Unicorn**
### Logits of this PR
1. Enhance the stability of the app by removing angular server, now serving static files using backend server instead
2. Change the appearance of user-interface to make it cleaner 

### Changes of this PR
1. Disable polygon fill color 
2. Remove sidebar (comment out)
3. Reduce default zoom level to enlarge viewport 
4. Config router for static frontend
5. Change backend port 
6. Build static frontend with the new port, but the environment file stays unchanged.

### Future improvement
1. Current frontend doesn't support build(ng build --prod) with production mode (not capable with building in AOT(ahead of time) mode), we need to modify codebase to fulfill this requirement, the changes can be many, so I leave them in next PR.

### CODE REVIEW

File: backend/web/__init__.py [hunk 1]
```diff
@@ -41,4 +41,4 @@ def create_app(test_config=None):
 
 if __name__ == '__main__':
     server_app = create_app()
-    server_app.run(debug=True)
+    server_app.run(port=2333, debug=True)
```
Comment (Yicong-Huang) [inline backend/web/__init__.py]: can this be left not changed?

File: backend/web/static/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>WildfiresFrontend</title>
```
Comment (Yicong-Huang) [inline backend/web/static/index.html]: change the title to be `Wildfires Map`

**Review by Yicong-Huang — CHANGES_REQUESTED** [review]
Some minor changes required.

I really do not want to expose port 2333 in the git history of master branch, can you reset the port to be default?

Also to ease the deployment, maybe we do not need to commit all the static files (since they will be build each time differently). Can you instead leave a bash file in the repo root, so it can do the following:
1. take in the host and port as command line parameters;
2. generate static frontend files with the given host and port;
3. mv the generated static frontend files to backend static/ directory (or just generate the static files to static/ directory directly);
4. start the backend server. 

Thank you

**Comment by Yicong-Huang** [comment]
fix  #3

**Comment by Yicong-Huang** [comment]
is this PR ready for review?

**Comment by AC-Unicorn** [comment]
Currently, we still can't build in production mode ( Or aot mode) since the leaflet error still exists.
The reason is unclear. So I changed build command in "Package.json" back to default build mode. 
I think we can merge the PR right now, and I'll do more research about the error, since the reason might be non-trivial.

**Comment by Yicong-Huang** [comment]
I've just tested on server with host changed to `0.0.0.0`. it still gives me the error 
```
ReferenceError: Can't find variable: L
```

**Comment by AC-Unicorn** [comment]
> I've just tested on server with host changed to `0.0.0.0`. it still gives me the error
> 
> ```
> ReferenceError: Can't find variable: L
> ```

Did you use "Empty Cache and Hard Reload"? assuming you are using chrome.

**Comment by Yicong-Huang** [comment]
OK that works. but it still cannot talk to backend. please change the production environment to be `host='0.0.0.0'` in frontend

**Comment by Yicong-Huang** [comment]
interesting, it still shows that 

```[Error] Failed to load resource: Could not connect to the server. (tweet-count, line 0)http://0.0.0.0:2333/data/recent-temp```

**Comment by AC-Unicorn** [comment]
> interesting, it still shows that
> 
> `[Error] Failed to load resource: Could not connect to the server. (tweet-count, line 0)http://0.0.0.0:2333/data/recent-temp`

Is the backend server running on 2333?

**Comment by Yicong-Huang** [comment]
yes, the backend is severing on 2333. 

and I can get response with 
`http://wildfires.ics.uci.edu:2333/data/recent-temp`

**Comment by Yicong-Huang** [comment]
I've manually changed the `environment.ts`
for the host to be `wildfires.ics.uci.edu` and it is working now.

However we'd better fix the prod mode soon.

**Comment by AC-Unicorn** [comment]
By adding a statement in polyfill.js, now the frontend can be compiled in production mode.
Angular will build the project into two bundles, one uses ES2015 syntax another one uses ES5 syntax which is older. To be compatible with all browsers, Polyfills in angular are few lines of code which make your application compatible for different browsers.

**Comment by Yicong-Huang** [comment]
I will postpone merging this PR as it is a relatively hacky way. the online demo is using this branch now.

**Comment by Yicong-Huang** [comment]
fixed #3

**Pull request merged**